app/code/Magento/Bundle/Test/Unit/Pricing/Price/ConfiguredPriceTest.php

```diff
@@ -0,0 +1,182 @@
+     */
+    private $amountInterfaceMock;
+    /**
+     * @var DiscountCalculator|MockObject
+     */
+    private $discountCalculator;
+
+    /**
+     * Initialize base dependencies
+     */
+    protected function setUp(): void
+    {
+        $basePrice = $this->getMockForAbstractClass(PriceInterface::class);
+        $basePrice->expects($this->any())->method('getValue')->willReturn($this->basePriceValue);
+
+        $this->priceInfo = $this->createMock(Base::class);
+        $this->priceInfo->expects($this->any())->method('getPrice')->willReturn($basePrice);
+        $this->product = $this->getMockBuilder(Product::class)
+            ->setMethods(['getPriceInfo', 'getOptionById', 'getResource', 'getId'])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->product->expects($this->once())->method('getPriceInfo')->willReturn($this->priceInfo);
+        $this->product->expects($this->any())->method('getId')->willReturn(123);
+
+        $this->item = $this->getMockBuilder(ItemInterface::class)
+            ->getMock();
+        $this->item->expects($this->any())->method('getProduct')->willReturn($this->product);
+
+        $this->priceCurrencyMock = $this->getMockForAbstractClass(PriceCurrencyInterface::class);
+
+        $this->jsonSerializerMock = $this->getMockBuilder(Json::class)
+            ->getMock();
+        $this->configuredPriceSelectionMock = $this->getMockBuilder(ConfiguredPriceSelection::class)
+            ->setMethods(['getSelectionPriceList'])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->configuredPriceSelectionMock->expects($this->any())->method('getSelectionPriceList')
+            ->willReturn($this->prepareAndReturnSelectionPriceDataStub());
+        $this->amountInterfaceMock = $this->getMockBuilder(AmountInterface::class)->getMock();
+        $this->amountInterfaceMock->expects($this->any())->method('getBaseAmount')
+            ->willReturn(100.0);
+        $this->calculator = $this->getMockBuilder(Calculator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->calculator->expects($this->any())->method('calculateBundleAmount')
+            ->willReturn($this->amountInterfaceMock);
+        $this->discountCalculator = $this->getMockBuilder(DiscountCalculator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->discountCalculator->expects($this->any())->method('calculateDiscount')
+            ->willReturn(-5.0);
+        $this->model = new ConfiguredPrice(
+            $this->product,
+            1,
+            $this->calculator,
+            $this->priceCurrencyMock,
+            null,
+            $this->jsonSerializerMock,
+            $this->configuredPriceSelectionMock,
+            $this->discountCalculator,
+        );
+        $this->model->setItem($this->item);
+    }
+
+    private function prepareAndReturnSelectionPriceDataStub()
```

Comment:
Please move the private method after the public ones. Also provide the method's docblock with a short description.
```suggestion
    private function prepareAndReturnSelectionPriceDataStub(): array
```